generators/spring-cloud/generators/gateway/command.ts

```diff
@@ -21,6 +21,11 @@ import type { JHipsterCommandDefinition } from '../../../base/api.js';
 const command: JHipsterCommandDefinition = {
   configs: {
     routes: {
+      description: 'Gateway manually configured routes',
```

Comment:
I'm unsure if the description is ever shown, but "Manually configured gateway routes" sounds better.

Reply:
```suggestion
      description: 'Manually configured gateway routes',
```

Reply:
If we drop the hide from cli, it will be shown.
Since this is only used in mvc gateway and it’s not clear if we want to support officially. It’s hidden for now.